fyne-io/fynedesk
# Tell other apps what DPI we are emulating

This means that apps that otherwise ignore environment (like GoLand)
will have a better chance of scaling correctly, if they support it.

When you load the desktop (or change the scale settings) we set the xrandr dpi property.
This is not foolproof, but it seems to hint at various apps what size to use.
These apps don't monitor changes to DPI sadly, but it's an improvement.

To test:
* Install etc
* Load a fresh X session
* open GoLand
* see it has a sensible font size
* quit it
* Change your scale preference in settings
* load GoLand and see that it has adapted.
